app/models/roster_spot.rb

```diff
@@ -0,0 +1,6 @@
+class RosterSpot < ApplicationRecord
+
+  belongs_to :team
```

Comment:
💡- When these are `create`ed, we can check to see if we've already hit the max # available per team.

Maybe something like this would be cool:

```ruby
class RosterSpot < ApplicationRecord
  # ...
  before_create :check_max_roster_limit
  # ...
  private
  
  def check_max_roster_limit
    errors.add(:team, 'Roster limit has been reached') if team.roster_spots + 1 > Team::ROSTER_LIMIT
  end
end
```